db/migrate/20230120144741_create_user.rb

```diff
@@ -1,12 +1,14 @@
 class CreateUser < ActiveRecord::Migration[7.0]
-  def change
-    create_table :users do |t|
-      t.string :Name
-      t.string :Photo
-      t.text :Bio
-      t.integer :PostsCounter
+   def change
+      create_table :users do |t|
+         t.string :Name
+         t.string :Photo
+         t.text :Bio
+         t.integer :PostsCounter
+         t.datetime :UpdatedAt
+         t.datetime :CreatedAt
```

Comment:
- [ ] For the correct notation and to avoid future conflicts please 🙏🏽 use snake_case for naming the columns e.g. `name`